### DISCUSSION TITLE
Transferring License

### DISCUSSION THREAD

**Discussion by sarpuser** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1366] [Q&A]
I remember seeing that you can activate a license up to three times. Does that count transferring licenses? Is there a way to deactivate a license so I can activate it on a new machine?

**Comment by MatthiasKerbl** [comment]
Hi @sarpuser,

Yes, you can activate your license up to three times. To deactivate a license on one of your devices, open the AlDente settings and go to General/Manage License/Deactivate License.

Best,
Matthias
